mods/cnc/chrome/assetbrowser.yaml

```diff
@@ -105,6 +105,13 @@ Container@ASSETBROWSER_PANEL:
 							Width: PARENT_RIGHT
 							Height: PARENT_BOTTOM
 							AspectRatio: 1
+						Label@ERROR:
+							X: 5
+							Width: 490 - 10
+							Height: 325
+							Align: Center
+							Visible: false
+							Text: Error displaying {{filename}} check debug.log
```

Comment:
Probably something to do with named values but looking at this again I think my mind was still in Python mode.